packages/f-header/src/components/Navigation.vue

```diff
@@ -86,7 +86,7 @@
                 </li>
 
                 <li
-                    :class="['c-nav-list-item has-sublist', { 'is-hidden': !userInfo || !showLoginOrUserInfo, 'open': navIsOpen }]"
+                    :class="['c-nav-list-item has-sublist', { 'is-hidden': !userInfo || !showLoginInfo, 'open': navIsOpen }]"
```

Comment:
NIT: this is a little hard to read, I'd suggest wrapping the object properties onto new lines.

```suggestion
                    :class="['c-nav-list-item has-sublist', {
                        'is-hidden': !userInfo || !showLoginInfo,
                        'open': navIsOpen
                    }]"
```